README.md

```diff
@@ -1 +1,185 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - CSS
+  - Github workflow
+  - Linters
+  - HTML
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Home page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+ - Vs code
+ - git
+
+
+### Setup
+
+Clone this repository to your desired folder:
+If you want to clone this repository, clone into your desired folder and enter "https://github.com/Blessman-Newton/Hello_Microverse-.git into your terminal
+
+### Install
+
+linters
+Html
+
+### Usage
+
+To run the project, execute the following command:
+- Live Server
+
+Open index.html in your browser
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "**/*.{css,scss}
+- hint.x
+
+### Deployment
+
+You can deploy this project using by transfering files in your local repository onto your saver.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="**Blessman-Newton">Blessman Newton</a>
+
+> Mention all of the collaborators of this project.
```

Comment:
- [ ] [OPTIONAL] This wasn't mentioned by the previous reviewer but please make sure to remove section descriptions (these are texts in your README that begin with ">") as they are only there to guide you the project owner on what to place on each section and should be removed. They look like the ones below
  
![firefox_agqYZoyjIO](https://user-images.githubusercontent.com/104968236/235629694-e4c57e8d-8f27-47f1-a5ff-330a7c700bda.png)
![firefox_JFmLUCfur3](https://user-images.githubusercontent.com/104968236/235629705-1d1fa459-fc8d-4fe3-96b8-a98e76c128cd.png)
![firefox_PtyYkhILG4](https://user-images.githubusercontent.com/104968236/235629707-4b8b008d-04a8-4290-bdb8-50e2367359e9.png)
![firefox_4iC6BjPRiR](https://user-images.githubusercontent.com/104968236/235629711-d3ad2761-a270-4c54-a51a-1afccdb938a4.png)
![firefox_n3DQDPc7az](https://user-images.githubusercontent.com/104968236/235629714-3b73c785-fbeb-45cd-9890-9278000d7780.png)
- So please make sure to remove the above texts from your readme.